sycl/source/detail/kernel_program_cache.hpp

```diff
@@ -130,6 +130,12 @@ class KernelProgramCache {
     MKernelFastCache.emplace(CacheKey, CacheVal);
   }
 
+  void reset() {
```

Comment:
A comment saying that this method is for testing purposes only might be useful here.